sys/cheri/cheri_exec.c

```diff
@@ -92,6 +96,12 @@ cheri_sigcode_capability(struct thread *td)
 	KASSERT(PROC_HAS_SHP(p),
 	    ("CheriABI requires shared page for sigcode"));
 
+	if (SV_PROC_FLAG(td->td_proc, SV_UNBOUND_PCC))
+		return cheri_capability_build_user_code(td,
+		    CHERI_CAP_USER_CODE_PERMS, CHERI_CAP_USER_CODE_BASE,
+		    CHERI_CAP_USER_CODE_LENGTH,
+		    PROC_SIGCODE(p) - CHERI_CAP_USER_CODE_BASE);
```

Comment:
```suggestion
		return (cheri_capability_build_user_code(td,
		    CHERI_CAP_USER_CODE_PERMS, CHERI_CAP_USER_CODE_BASE,
		    CHERI_CAP_USER_CODE_LENGTH,
		    PROC_SIGCODE(p) - CHERI_CAP_USER_CODE_BASE));
```
Unless there's been a style(9) relaxation here.

Reply:
Oops